src/scss/sections/_howto.scss

```diff
@@ -1,22 +1,38 @@
 .how-to {
-    background-color: $primary-light;
+	background-color: $primary-light;
 
-    .carousel--centered {
-        overflow-x: hidden;
-    }
+	.carousel--centered {
+        position: initial;
 
-    .carousel-control-next {
-        background: linear-gradient(270deg, $primary-light 0%, $primary-light 10%, rgba(255, 255, 255, 0) 99%, rgba(255, 255, 255, 0) 100%);
+        @include media-breakpoint-up(lg) {
+            max-width: 64em;
+            margin: 0 auto;
+        }
     }
 
-    .carousel-control-prev {
-        background: linear-gradient(90deg, $primary-light 0%, $primary-light 10%, rgba(255, 255, 255, 0) 99%, rgba(255, 255, 255, 0) 100%);
-    }
+	.carousel-control-prev,
```

Comment:
carousel controls on desktop are not centred vertically, on mobile after clicking on right control the control shifts upper, please review